libraries/Microsoft.Bot.Schema/SpeechConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema
+{
+    /// <summary>
+    /// Defines constants that can be used in the processing of speech interactions.
+    /// </summary>
+    public static class SpeechConstants
+    {
+        // The xml tag structure to indicate an empty speak tag, to be used in the 'speak' property of an Activity. When set this indicates to the channel that speech should not be generated.
```

Comment:
Could you please convert this comment to XML format? It will be needed when we remove the CS1591 warnings.